src/pipecat/services/openai_realtime/context.py

```diff
@@ -4,18 +4,15 @@
 # SPDX-License-Identifier: BSD 2-Clause License
 #
 
-"""OpenAI Realtime LLM context and aggregator implementations."""
+"""OpenAI Realtime LLM context and aggregator implementations.
 
-import warnings
+.. deprecated:: 0.0.91
```

Comment:
Even though we're changing this file in this PR, it was technically deprecated last version, hence 0.0.91.